src/utils.js

```diff
@@ -8,39 +10,77 @@ const getModels = (request) => {
   const noRequestModels = !request.models;
   if (noGetDb && noRequestModels) {
     return notImplemented('`request.getDb` or `request.models` are not defined.'
-                   + 'Be sure to load hapi-sequelize before hapi-sequelize-crud.');
+      + 'Be sure to load hapi-sequelize before hapi-sequelize-crud.');
   }
 
   const { models } = noGetDb ? request : request.getDb();
 
   return models;
 };
 
-export const parseInclude = request => {
+export const parseInclude = async(request) => {
```

Comment:
because **getModelInstance** may loop itself, for example i got object like this:
```
{
  include: {
   model: Team,
   include: {
     model: Player,
     where: {
       name: Pinot
     },
     include: {
       model: Master,
       as: 'Coach',
       where: {
         name: 'Shifu'
       }
     }
   }
  }
}
```
the **getModelInstance** function will loop through all of its **include** children to convert each model name string to be its model instance, notice that conversion here needs logic like looping all existing models to find the right model instance